src/test/java/spoon/test/parent/ContractOnSettersParametrizedTest.java

```diff
@@ -220,6 +223,35 @@ private void testContract(CtType<?> toTest) throws Throwable {
 				int nBefore = changeListener.nbCallsToOnAction;
 				changeListener.changedElements = new ArrayList<>();
 
```

Comment:
Could you add a fat comment of what you do here and why we need it?

It's unclear how this relates to the problem described in the PR description.

Thanks!